src/Component/RequestBuilder/FacetSet.php

```diff
@@ -108,28 +118,31 @@ public function buildComponent($component, $request)
      *
      * @param Request    $request
      * @param FacetField $facet
+     * @param compatible $compatible false if local parameters should be used
      */
-    public function addFacetField($request, $facet)
+    public function addFacetField($request, $facet, $compatibility_mode = true)
```

Comment:
I prefer to name the variable $use_local_params and to invert the logic. So the default would be false.